examples/api/lib/widgets/animated_list/animated_list.0.dart

```diff
@@ -50,11 +50,11 @@ class _AnimatedListSampleState extends State<AnimatedListSample> {
     );
   }
 
-  // Used to build an item after it has been removed from the list. This method
-  // is needed because a removed item remains visible until its animation has
-  // completed (even though it's gone as far as this ListModel is concerned).
-  // The widget will be used by the [AnimatedListState.removeItem] method's
-  // [AnimatedListRemovedItemBuilder] parameter.
+  /// Used to build an item after it has been removed from the list. This method
+  /// is needed because a removed item remains visible until its animation has
+  /// completed (even though it's gone as far as this ListModel is concerned).
+  /// The widget will be used by the [AnimatedListState.removeItem] method's
+  /// [AnimatedRemovedItemBuilder] parameter.
```

Comment:
I think '//' is probably more appropriate here because as it stands we probably wouldn't want the exact text here as dartdocs (for example, it doesn't have a leading line that says what it is). As it stands, it's a comment (a good one, that explains what the example is about).

Reply:
I hear what you're saying, and I know what the style guide says, but consider these three things:

1) if they are dartdoc comments, then the `[AnimatedRemovedItemBuilder]` becomes a link that could be quite helpful to the user to click on in an IDE. Otherwise, it just looks like a piece of text that wishes it were a link.
2) This is in the examples, which aren't actually processed by dartdoc.
3) It's on a private interface that, even if the examples were processed by dartdoc, would not show up in the output.

That said, I'm happy to either spruce up the comment so it's "good enough" to be a dartdoc comment, or remove the brackets and make it a "regular" comment that doesn't try to link to the symbols.

Reply:
I went with "sprucing it up"...